lib/mayaUsd/fileio/jobs/writeJob.cpp

```diff
@@ -640,12 +634,19 @@ bool UsdMaya_WriteJob::_FinishWriting()
         primWriterLoop.loopAdvance();
     }
 
+    _extrasPrimsPaths.clear();
+
     // Run post export function on the chasers.
     MayaUsd::ProgressBarLoopScope chasersLoop(mChasers.size());
     for (const UsdMayaExportChaserRefPtr& chaser : mChasers) {
         if (!chaser->PostExport()) {
             return false;
         }
+
+        // Collect extra prims paths from chasers
+        for (const SdfPath& path : chaser->GetExtraPrimsPaths()) {
```

Comment:
I think there is a way to insert a range in C++ std::vector? Not super important.